frontend/nginx.conf

```diff
@@ -0,0 +1,30 @@
+pid /tmp/nginx.pid;
+worker_processes  auto;
+error_log  /var/log/nginx/error.log notice;
+
+http {
+    proxy_temp_path /tmp/proxy_temp;
+    client_body_temp_path /tmp/client_temp;
+    fastcgi_temp_path /tmp/fastcgi_temp;
+    uwsgi_temp_path /tmp/uwsgi_temp;
+    scgi_temp_path /tmp/scgi_temp;
+    include /etc/nginx/mime.types;
+    
+    default_type application/octet-stream;
+
+    server_tokens off;
+    
+    server {
+        listen       3000;
```

Comment:
I wonder if using `listen 127.0.0.1:3000` would help with the port mapping problem?